src/main/java/io/jenkins/plugins/OpsLevelGlobalConfigUI.java

```diff
@@ -0,0 +1,120 @@
+package io.jenkins.plugins;
```

Comment:
As mentioned, I think there might be a nicer spot for this code than in a separate `RunListener`, but for now if it works, it works.